api/docker/web/Dockerfile

```diff
@@ -8,20 +12,17 @@ WORKDIR /www/data
 RUN apk --no-cache upgrade
 RUN apk --no-cache add wget libcap
 
-# Add Confd to configure nginx on start
-ENV CONFD_VERSION="0.16.0"
-ADD https://github.com/kelseyhightower/confd/releases/download/v${CONFD_VERSION}/confd-${CONFD_VERSION}-linux-${PLATFORM} /usr/local/bin/confd
-RUN chmod +x /usr/local/bin/confd
+COPY --chown=nginx api/docker/web/config/default.conf /etc/nginx/conf.d/default.conf.tmpl
+COPY --chown=nginx api/docker/web/config/nginx.conf /etc/nginx/nginx.conf
 
-COPY --chown=nginx client/docker/web/confd /etc/confd
-COPY --chown=nginx client/resources/public/ /www/data/public/
+RUN envsubst '$APP_HOST $APP_PORT' < /etc/nginx/conf.d/default.conf.tmpl > /etc/nginx/conf.d/default.conf
```

Comment:
The `X < Y > Z` syntax is a bit unintuitive. I personally prefer something more like `cat Y | envsubst X > Z` as it feels a bit more intuitive (left to right). But maybe that's just me.

Reply:
Hmm yeah you might be right there. I'll probably change it when I do the API side next